app/Http/Controllers/ImportController.php

```diff
@@ -193,6 +193,9 @@ public function server(Request $request)
 				case 't':
 					$this->memLimit *= 1024 * 1024 * 1024 * 1024;
 					break;
+				default:
```

Comment:
Does this do anything?

Reply:
nope. But sonar was complaining :sweat_smile:

Reply:
Could you not make it just `break;'? Why times it by 1??